zappa/handler.py

```diff
@@ -336,9 +336,24 @@ def handler(self, event, context):
 
         # This is a direct command invocation.
         elif event.get('command', None):
-
             whole_function = event['command']
+            args = []
+            kwargs = {}
```

Comment:
@xuru no worries! And thank you for working on this. I'm going to tinker with it probably tomorrow as well.